New log file opened at Sat May 02 10:42:04 PDT 2015 by 29646@Thomass-MacBook-Pro.local
Taking next connection to: 58575
New client attempting to connect on port: 58576-58575
  Address = /10.0.0.16:58575
This client on port 58576 has uuid e13c5e73-1415-4103-9864-24c7bab9a253 configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-e13c5e73-1415-4103-9864-24c7bab9a253
Taking next connection to: 58575
New client attempting to connect on port: 58577-58575
  Address = /10.0.0.16:58575
Handshake exception on socket: 58577
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(SocketInputStream.java:189)
	at java.net.SocketInputStream.read(SocketInputStream.java:121)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 58575
New client attempting to connect on port: 58578-58575
  Address = /10.0.0.16:58575
This client on port 58578 has uuid 66b27298-e6a8-4d59-aa2a-61971f7feb58 configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-e13c5e73-1415-4103-9864-24c7bab9a253, activator-66b27298-e6a8-4d59-aa2a-61971f7feb58
Taking next connection to: 58575
New client attempting to connect on port: 58583-58575
  Address = /10.0.0.16:58575
Handshake exception on socket: 58583
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.io.EOFException
	at java.io.DataInputStream.readInt(DataInputStream.java:392)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 58575
New client attempting to connect on port: 58584-58575
  Address = /10.0.0.16:58575
This client on port 58584 has uuid e1238306-db5a-4c4f-81f2-22547f159359 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-66b27298-e6a8-4d59-aa2a-61971f7feb58, play-fork-run-e1238306-db5a-4c4f-81f2-22547f159359
Taking next connection to: 58575
